Add intro talks 2023 (#37)
kitctf/www: _posts/2023-04-19-into.md

---
layout: post
title: "Intro Talks 2023"
categories: learning
permalink: /intro/
author: mawalu
---

> Starting April 27 we will have introduction talks for the most important categories of Capture the flag.

Reviewer: ```suggestion
Starting on the 27th of April, we will be holding introductory talks for the main categories of Capture the Flag.
```

> We meet Thursdays at 7pm in the CS building [50.34](https://www.kit.edu/campusplan/) room -120 (sometimes we may be in room -118).

Reviewer: ```suggestion
We meet on Thursdays at 7pm in the CS building [50.34](https://www.kit.edu/campusplan/) room -120 (sometimes we may meet in room -118).
```

>  * 27.4.23: What are CTFs? & Web Security

Reviewer: ```suggestion
 * 27.04.23: What are CTFs? & Web Security
```

>  * 4.5.23: Reverse Engineering

Reviewer: ```suggestion
 * 04.05.23: Reverse Engineering
```

>  * 11.5.23: Binary Exploitation

Reviewer: ```suggestion
 * 11.05.23: Binary Exploitation
```

>  * 25.5.23: Cryptography

Reviewer: ```suggestion
 * 25.05.23: Cryptography
```
